> for `rs274/options.py‎` rename isn't needed, expose EMC2_TCL_DIR via the already-generated nf.py and we could keep `options.py` static, just read it:
> 
> ```
> rip = os.path.join(nf.PREFIX, 'tcl', 'linuxcnc.tcl')
> LINUXCNC_TCL = rip if os.path.exists(rip) else os.path.join(nf.EMC2_TCL_DIR, 'linuxcnc.tcl')
> ```
> 
> then can cleanup
> 
> * the new `AC_CONFIG_FILES(../lib/python/rs274/options.py)` line
> * the .gitignore entry.
> 
> It works either way, but it might be easier to keep track of things for you?

latency-test: `module 'nf' has no attribute 'EMC2_TCL_DIR'`

Also, build bot error:

`configure: error: could not determine Tcl package dir from tcl_pkgPath`

I might have done something wrong.. Due to how m4 handles syntax, the proper way to do the braces is the one I just pushed, it's a slight modification to what you had but it won't work as-is, it needs to be double bracketed.

--- a/debian/control.top.in
+++ b/debian/control.top.in
@@ -42,6 +42,7 @@ Build-Depends:
     python3-dev,
     python3-tk,
     python3-xlib,
+    tcl,
     tcl@TCLTK_VERSION@-dev,
     tclx,
     tk@TCLTK_VERSION@-dev,

--- a/lib/python/nf.py.in
+++ b/lib/python/nf.py.in
@@ -105,6 +105,7 @@ def find_prefix(f):
     return find_prefix(os.path.dirname(f))
 
 PREFIX = "@prefix@"
+EMC2_TCL_DIR = "@EMC2_TCL_DIR@"
 SHARE = os.path.join(PREFIX, "share", "axis")
 tcl_libdir = os.path.join(SHARE, "tcl")
 

--- a/src/configure.ac
+++ b/src/configure.ac
@@ -735,7 +735,6 @@ else
     EMC2_SCRIPT=$EMC2_BIN_DIR/linuxcnc
     EMC2_SUFFIX=""
     EMC2_ICON=linuxcncicon
-    EMC2_TCL_DIR="${TCL_GET_DIR}/linuxcnc"
     EMC2_TCL_LIB_DIR="${EMC2_TCL_DIR}"
     EMC2_LANG_DIR="${EMC2_TCL_DIR}/msgs"
     EMC2_PO_DIR=${prefix}/share/locale

